module/move/wca/src/ca/commands_aggregator/aggregator.rs

```diff
@@ -237,10 +237,28 @@ pub( crate ) mod private
     {
       let program = program.as_ref();
 
+      let mut args: Vec< String > = program
+      .split_whitespace()
+      .map( | s | s.to_string() )
+      .collect();
+
+      for i in 0..args.len()
+      {
+        let path = std::path::Path::new( &args[ i ] );
+        if path.is_dir() && !(path == std::path::Path::new("."))
```

Comment:
Format

Reply:
What happens if a subject is not a path but contains `:`?
E.g.
```
.echo "details: about something"
```
In addition, if this is a file path (and not a directory), your solution returns an error